cypress/support/fragments/inventory/inventoryViewSource.js

```diff
@@ -11,8 +12,11 @@ export default {
   notContains:(notExpectedText) => {
     cy.expect(HTML(including(notExpectedText)).absent());
   },
-  waitLoading: () => {
-    cy.expect(HTML(including(title)).exists());
+  waitInstanceLoading: () => {
+    cy.expect(HTML(including(instanceTitle)).exists());
+  },
+  waitHoldingLoading: () => {
+    cy.expect(HTML(including(holdingTitle)).exists());
```

Comment:
May be better to create one function with parameter title?

```
waitLoading: title => {
    cy.expect(HTML(including(title)).exists());
  },
```
or parameter could be page name:

```
waitLoading: pageName => {
    const title = pageName.title
    cy.expect(HTML(including(title)).exists());
  },
```

Reply:
in this case you will need to provide right strings in test   or specify somewhere special related constants with reusing into tests.... I can't see benefits from your suggestion, may be I've lost something